src/plugins/Network.js

```diff
@@ -697,4 +705,22 @@ module.exports = class Network extends OverlayPlugin {
             }
         }
     }
+
+    /**
+     * Update mobile signal Detail information.
+     *
+     * @param {string} detail Signal detail to update.
+     * @param {string} value  New signal detail value.
+     * @param {string} reset  If true ignore value and set "".
+     */
+     updateDetail(detail, value, reset) {
+        if(! detail) {
```

Comment:
```suggestion
        if(!detail) {
```
code style. But this should be handled by the linter ? You can try running `yarn validate` ?

Reply:
`yarn lint --fix` will fix some of the formatting issues too :wink:

Reply:
I think the `detail` argument will always be a defined string, no? Do we need this check `if(!detail)`?

Reply:
yarn validate
yarn run v1.22.19
$ yarn lint; yarn checkstyle
$ eslint src tests gulp
$ eslint -f checkstyle -o ./tests/reports/lint/report.xml src tests gulp
Done in 3.77s.